.idea/markdown-navigator.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file was probably added before .gitignore was modified. .idea folder should be removed from the repository.